priv/sites/testsandbox/config

```diff
@@ -10,7 +10,11 @@
  %% Hostname for the test sandbox. Do NOT change this! It will break the tests.
  {hostname, "localhost:8040"},
 
- {dbdatabase, none},
+ {dbhost, "127.0.0.1"},
+ {dbport, 5432},
+ {dbuser, "postgres"},
+ {dbpassword, ""},
+ {dbdatabase, "testsandbox"},
```

Comment:
As there are no added (database) tests in this PR, I don’t see the benefit of changing the sandbox setup here. This PR is about rebar3, let’s leave an improved test setup to #1235.